sdcm/utils/aws_utils.py

```diff
@@ -268,8 +269,10 @@ def get_ec2_network_configuration(regions: list[str], availability_zones: list[s
     ec2_subnet_ids = []
     for region in regions:
         aws_region = AwsRegion(region_name=region)
+        availability_zones = [region + availability_zone for availability_zone in availability_zones]
```

Comment:
can you revert this changes ? they don't seem to do anything. just adding a print.

Reply:
@yarongilor please revert those unneeded changes